src/main/resources/jupyter/cluster-site.conf

```diff
@@ -24,6 +24,19 @@
     ProxyPass / http://127.0.0.1:8001/ retry=10
     ProxyPassReverse / http://127.0.0.1:8001/
 
+    # Welder
```

Comment:
This is copy/paste and I don't understand it well.  I'd appreciate a review from Leo engineers who have worked on this previously.

Reply:
it looks correct to me. i've commented inline to provide clarity of what it all means

```
    RewriteCond %{HTTP:Upgrade} !=websocket     //if the request is NOT a websocket request
    RewriteCond %{REQUEST_URI} /proxy/[^/]*/[^/]*/welder/.* [NC]     //if the request URI matches the regex, ignoring case sensitivity (NC = not case sensitive)
    RewriteRule /proxy/[^/]*/[^/]*/welder/(.*) http://127.0.0.1:8080/$1 [P,L] //if all of the above conds are true, rewrite the request to "http://127.0.0.1:8080/$1", where $1 is the first capture group and so on. P = proxy the request, L = "last", which means to stop processing any following rulesets
```